package com.join.mgps.activity;

import android.content.Context;
import android.widget.TextView;
import com.BaseActivity;
import com.join.android.app.mgsim.wufun.R;
import com.join.mgps.Util.IntentDateBean;
import com.join.mgps.Util.IntentUtil;
import com.join.mgps.Util.RequestBeanUtil;
import com.join.mgps.dto.RecomDatabean;
import com.join.mgps.dto.ResultMainBean;
import java.util.List;
import org.androidannotations.annotations.AfterViews;
import org.androidannotations.annotations.Background;
import org.androidannotations.annotations.Click;
import org.androidannotations.annotations.EActivity;
import org.androidannotations.annotations.UiThread;
import org.androidannotations.annotations.ViewById;
@EActivity(R.layout.downlarge_game_dialog_activity)
/* loaded from: classes4.dex */
public class DownLargeGameDialogActivity extends BaseActivity {
    @ViewById

    /* renamed from: b  reason: collision with root package name */
    TextView f29955b;
    @ViewById

    /* renamed from: c  reason: collision with root package name */
    TextView f29956c;
    @ViewById

    /* renamed from: d  reason: collision with root package name */
    TextView f29957d;

    /* renamed from: e  reason: collision with root package name */
    IntentDateBean f29958e;

    /* renamed from: f  reason: collision with root package name */
    com.join.mgps.rpc.e f29959f;

    /* renamed from: g  reason: collision with root package name */
    Context f29960g;

    /* JADX INFO: Access modifiers changed from: package-private */
    @AfterViews
    public void afterviews() {
        this.f29960g = this;
        this.f29959f = com.join.mgps.rpc.impl.d.P1();
        IntentDateBean intentDateBean = new IntentDateBean();
        this.f29958e = intentDateBean;
        intentDateBean.setLink_type(4);
        this.f29958e.setLink_type_val("http://anv3cjapi.5fun.com/member/vip_view/welcome?go=vip");
        getData();
    }

    /* JADX INFO: Access modifiers changed from: package-private */
    @Click
    public void g0() {
        finish();
    }

    /* JADX INFO: Access modifiers changed from: package-private */
    @Background
    public void getData() {
        try {
            ResultMainBean<List<RecomDatabean>> o02 = this.f29959f.o0(RequestBeanUtil.getInstance(this.f29960g).getSimulatorRequest("", "", 0));
            if (o02 == null || o02.getMessages() == null || o02.getMessages().getData() == null || o02.getMessages().getData().size() <= 0) {
                return;
            }
            RecomDatabean recomDatabean = o02.getMessages().getData().get(0);
            this.f29958e = recomDatabean.getSub().get(0).getIntentDataBean();
            i0(recomDatabean.getMain().getTitle(), recomDatabean.getMain().getSub_title());
        } catch (Exception e5) {
            e5.printStackTrace();
        }
    }

    /* JADX INFO: Access modifiers changed from: package-private */
    @Click
    public void h0() {
        IntentUtil.getInstance().intentActivity(this.f29960g, this.f29958e);
        finish();
    }

    /* JADX INFO: Access modifiers changed from: package-private */
    @UiThread
    public void i0(String str, String str2) {
        this.f29955b.setText(str);
        this.f29956c.setText(str2);
    }
}
